hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/RootedOzoneFileSystem.java

```diff
@@ -135,6 +137,14 @@ public boolean hasPathCapability(final Path path, final String capability)
     if (cap) {
       return cap;
     }
+    // this switch is for features which are in the OFS client.
+    switch (validatePathCapabilityArgs(path, capability)) {
```

Comment:
Please put this CommonPathCapabilities.LEASE_RECOVERABLE check into OzonePathCapabilities.hasPathCapability. 

Besides this [RootedOzoneFileSystem.java](https://github.com/apache/ozone/pull/6638/files#diff-8e219ae2e7dcd13144bf5e888a9c34dc5cfecd34b68518d030943f10e29d35aa), there is another one with same name for ozonefs-hadoop3.

Reply:
Done.